packages/quick_actions/quick_actions_ios/ios/quick_actions_ios.podspec

```diff
@@ -14,9 +14,10 @@ Downloaded by pub (not CocoaPods).
   s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :http => 'https://github.com/flutter/plugins/tree/main/packages/quick_actions' }
   s.documentation_url = 'https://pub.dev/packages/quick_actions'
-  s.source_files = 'Classes/**/*'
+  s.source_files = 'Classes/**/*.{h,m}'
   s.public_header_files = 'Classes/**/*.h'
```

Comment:
This is making the test header a public header. We've made that mistake in other plugins, but we shouldn't replicate it here.

Reply:
@stuartmorgan Looks like the test header must be public. I got this error when not adding the test header to this `public_header_files`: 

```
    - ERROR | [iOS] xcodebuild:  /Users/admin/Library/Developer/Xcode/DerivedData/App-cgyuubhexmicjpawtwtidfnntejc/Build/Products/Debug-iphonesimulator/quick_actions_ios/quick_actions_ios.framework/Modules/module.modulemap:8:12: error: header 'FLTQuickActionsPlugin_Test.h' not found
```

If I understand this error correctly, `modulemap` file defines the module by mapping headers in the output directory. The `Test` submodule is no difference than regular module in terms of visibility, except that it's scoped under a submodule. 

CC @jmagman probably know more about modulemap.

Reply:
Please ignore my previous comment, I misunderstood how private headers work. It turned out that private headers ARE copied to the output directory (so that modulemap can refer to them).